types/jasmine/index.d.ts

```diff
@@ -838,10 +838,10 @@ declare namespace jasmine {
 
     interface CustomReporter {
         jasmineStarted?(suiteInfo: SuiteInfo): void;
```

Comment:
Both this and L845 seem to take in a Done parameter per https://jasmine.github.io/api/edge/Reporter.html

Reply:
nice catch :100: not sure why I missed that.